uiowa/uiowa
# International Programs migration updates

Follow-up updates prior to being run again in dev. Full testing instructions are below, but a code review may be acceptable as it will have another 'test' run in dev prior to running in prod.

(Also snuck in a minor comment update for `sitenow_find_text` that shows up in phpcs.)

# How to test


Sync d7 site in DevDesktop (on uiowa703)
```
git checkout ip-migration-updates
git pull
ddev blt ds --site=international.uiowa.edu
ddev drush @international.local en international_migrate -y 
ddev drush @international.local uli /admin/config/sitenow/sitenow-migrate
```
Complete form using the help text as a guide
`drush @international.local ms` to check that both migrations are registered
`drush @international.local mim --limit=200` <- repeat until done for each migration
While migrations are running, there should be relatively few "cannot replace XXXX link in article XXXX" messages
When last migration batch is run, there should not be a "Unable to update XXX, XXX, XXX, XXX pages" message
Spot check links in any of nodes 1376, 1391, 1396, 1411; each link on a person's name, if there is one, should go to the article about their receiving the award
Check the tags, make sure there aren't duplicates from running multiple migration batches
Spot check some articles and see that the tags have been migrated
Spot check some articles and see that images appear at an acceptable size, and appropriately aligned on the page, generally following the source alignment
